crt_portal/static/js/set_referrer_on_landing_page.js

```diff
@@ -0,0 +1,8 @@
+(function(root) {
+  referrer = document.referrer;
+  host = document.location.host;
+
+  if (localStorage.getItem('referrer') == null && referrer != '') {
```

Comment:
Would we want to overwrite localstorage if there is a new referrer?  I wonder if we should just do 

```if (referrer) {```